### ISSUE TITLE
Indicate value on layer opacity slider

### ISSUE THREAD

**Issue by eishiya** [mapeditor/tiled#3767]
This is a usability tweak suggested by a user on Discord: When setting layer opacity via the slider in the Layers view, it can be hard to tell what your actual value is if the Properties view shows something else at the moment. Plus, the Properties view might be on the other end of the screen.

Displaying the value as a tooltip while hovering or dragging the slider would help :D

**Comment by TechAtVishal** [comment]
I would like to work on this, please assign this to me.

**Comment by bjorn** [comment]
@TechAtVishal You're welcome to work on this, I'll assign the issue to you. :-)

**Comment by TechAtVishal** [comment]
@eishiya @bjorn when i fire up the latest tiled version 1.11.2, the slider is there but when i build it with qt the slider is simply not there in the Layers view box. Is this feature/tool removed or changed in the recent updates to the repo?

<img width="1918" height="662" alt="Image" src="https://github.com/user-attachments/assets/5155c11f-d1e3-4269-8b40-24c418ad07c6" />
<img width="1918" height="390" alt="Image" src="https://github.com/user-attachments/assets/cdc04b68-590a-4cb3-bc70-af4c770c8566" />

**Comment by bjorn** [comment]
@TechAtVishal Ah, it moved to the Properties view, where in fact the value is shown alongside the slider, so I guess there would not be much value in displaying the value in a tooltip as well.

> <img width="583" height="352" alt="Image" src="https://github.com/user-attachments/assets/c21f0df5-7f0d-4c02-85d1-13eb28e026b7" />

@eishiya Maybe we should just close this issue?

**Comment by TechAtVishal** [comment]
@bjorn yeah i saw that in the properties view, forgot to mention it in the previous comment. I guess it will be up with the new update, right?

**Comment by bjorn** [comment]
@TechAtVishal Yeah, this is one of the changes that will be released with Tiled 1.12.

**Comment by eishiya** [comment]
> [@eishiya](https://github.com/eishiya) Maybe we should just close this issue?

I recall saying before that I think the Layers view should retain its opacity slider, as that's the common place for it, even if it means there are two sliders in different places. Changing layer opacity isn't as common as toggling visibility or renaming layers (both of which can be done in both the Layers view and the properties view), but it still feels like a basic enough layer property to warrant the space it takes up.
And you know me, I'm almost always in favour of more GUI customisation, so I'd love for the slider to be something you can disable/enable in the Layer view's options xP

**Comment by bjorn** [comment]
Alright, of course I had not really expected your opinion to change regarding that, but I was a bit afraid of it since due to string freeze I'd rather not introduce an option for this at the moment.

But, we could re-introduce the slider without an option to remove it for now. Then the question is, should we re-introduce it the way it currently appears in the Properties view, with the number edit on the side? Or is this still a case where a slider with the value in a tooltip would be better?

**Comment by eishiya** [comment]
My first thought was that having an identical widget would be good since it adds another way to modify the value, but I do worry about the extra vertical space a number input box will probably require. If the option to hide the opacity input is forthcoming, then the extra height would be only a short-lived problem.

**Comment by TechAtVishal** [comment]
If the plan is to bring back that slider in Layers View box, please let me do it, that way I can learn about the codebase. :)

**Comment by bjorn** [comment]
@TechAtVishal Alright, please go ahead. The commit in which the slider was removed is 6bbae7f044aaca62cc30a128733339679492075d, so you can use that to bring it back and then add a value tooltip or try using the `IntProperty` to create the combined widget. :-)

**Comment by TechAtVishal** [comment]
Sorry for the long pause. I just created a pull request https://github.com/mapeditor/tiled/pull/4549 resolving this issue. Please go through it whenever you have some time.
